packages/controllers/src/snaps/SnapController.ts

```diff
@@ -148,6 +149,11 @@ export interface SnapRuntimeData {
    * @see {@link SnapController:constructor}
    */
   interpreter: StateMachine.Service<StatusContext, StatusEvents, StatusStates>;
+
+  /**
+   * The snap source code
+   */
+  sourceCode: null | string;
```

Comment:
Why use `null` here?

```suggestion
  sourceCode?: string;
```

Reply:
I just mirrored the other runtime properties, not sure why they are null

Reply:
Perhaps so it is harder to forget defining it in `.set({ ...})`?